src/test/java/StringClassTest.java

```diff
@@ -0,0 +1,67 @@
+import org.junit.jupiter.api.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class StringClassTest {
+
+    @Nested
+    @DisplayName("첫번째 요구사항 클래스")
+    class NumberTest {
+          private String NumbersData = "1,2";
+          private String NumberData = "1";
+          private String [] array;
+
+        @Test
+        @DisplayName(value = "1,2 split으로 분리여부 확인 테스트")
+        public void splitNumbersTest() {
+            array = NumbersData.split(",");
+            assertThat(array).contains("1","2");
```

Comment:
```suggestion
            assertThat("1,2".split(",")).contains("1", "2");
```

위와 같이 극단적으로 단순하게 구현하는 것은 어떨까?

Reply:
완료했습니다!